src/comm/TcpAcceptor.cc

```diff
@@ -302,7 +302,7 @@ Comm::TcpAcceptor::acceptOne()
         return;
     }
 
-    newConnDetails->nfmark = Ip::Qos::getNfmarkFromConnection(newConnDetails, Ip::Qos::dirAccepted);
```

Comment:
nfmark variable is responsible for packet marks. AFAIK there is no way to get packet mark in userspace (even if it was, it would be slow). Taking into account that packet and connection marks may differ and that we do not provide ACLs for packet mark checking, it will be better to leave the variable unset (zero).